app/shared/resource-map/MapContainer.js

```diff
@@ -76,4 +87,9 @@ UnconnectedResourceMapContainer.propTypes = {
   }),
 };
 
-export default connect(selector)(UnconnectedResourceMapContainer);
+UnconnectedResourceMapContainer.propTypes = {
+  ...UnconnectedResourceMapContainer.propTypes,
+  ...geoPropTypes,
```

Comment:
- [ ] Why not have these just in the same place where the other UnconnectedResourceMapContainer propTypes are declared?